src/getInfo.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable dot-notation */
+
+const packageJson = require('../package.json');
+
+module.exports = () => ({
+  type: packageJson.name,
+  version: packageJson.version,
+  dependencies: packageJson.compileDependencies,
```

Comment:
as discussed, I think removing the dependencies field is safe and useful as it will facilitate next step